graph/assemblygraph.cpp

```diff
@@ -1046,13 +1055,13 @@ int AssemblyGraph::buildDeBruijnGraphFromAsqg(QString fullFileName)
                     nodeName = "node";
                 nodeName += "+";
 
-                QByteArray sequence = lineParts.at(2).toLocal8Bit();
-                int length = sequence.length();
+                Sequence sequence{lineParts.at(2).toLocal8Bit()};
+                int length = static_cast<int>(sequence.size());
```

Comment:
maybe fix length in node ctor? Why it is signed?

Reply:
Maybe, the author followed Google Style Guide :)
https://google.github.io/styleguide/cppguide.html#Integer_Types